src/Uno.UWP/Graphics/Display/DisplayInformation.wasm.cs

```diff
@@ -152,6 +161,24 @@ private static DisplayOrientations ParseJsOrientation(string jsOrientation)
 				_ => DisplayOrientations.None,
 			};
 		}
+
+		private static string ToJsOrientation(DisplayOrientations orientations)
+		{
+			return orientations switch
+			{
+				DisplayOrientations.Landscape => "landscape-primary",
+				DisplayOrientations.LandscapeFlipped => "landscape-secondary",
+				DisplayOrientations.Portrait => "portrait-primary",
+				DisplayOrientations.PortraitFlipped => "portrait-secondary",
+				_ => "any"
+			};
+		}
+
+		private static Task SetOrientationAsync(DisplayOrientations orientations, CancellationToken ct)
+		{
+			var jsOrientation = WebAssemblyRuntime.EscapeJs(ToJsOrientation(orientations));
+			return WebAssemblyRuntime.InvokeAsync($"{JsType}.setOrientationAsync(\"{jsOrientation}\")", ct);
```

Comment:
Not sure if it's worth doing a check to call this only if `jsOrientation` is different than the existing old value

Reply:
To get the existing value we would need a Javascript interop call anyway.